# Handling Errors
try:
    print(x)
except NameError as ex:
    print("An Exception Occurred : " + str(ex))